### PR TITLE
Add CTA for play button

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR adds conditional display + `onClick` for the play CTA of the `AssistantPreview2` in the gallery variant.

### CODE REVIEW

File: sparkle/src/components/AssistantPreview2.tsx [hunk 1]
```diff
@@ -160,8 +161,15 @@ const ListVariantContent = (props: ListVariantAssistantPreviewProps) => {
 const GalleryVariantContent = (
   props: GalleryVariantAssistantPreviewProps & { isHovered: boolean }
 ) => {
-  const { renderActions, description, title, pictureUrl, subtitle, isHovered } =
-    props;
+  const {
```
Comment (philipperolet) [inline sparkle/src/components/AssistantPreview2.tsx:164]: could that be inlined in the function declaration?

File: sparkle/src/components/AssistantPreview2.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ type ListVariantAssistantPreviewProps = BaseAssistantPreviewProps & {
 type GalleryVariantAssistantPreviewProps = BaseAssistantPreviewProps & {
   variant: "gallery";
   renderActions?: (isHovered: boolean) => React.ReactNode;
+  onPlayClick?: (e: SyntheticEvent) => void;
```
Comment (philipperolet) [inline sparkle/src/components/AssistantPreview2.tsx:30]: why pass a syntheticevent?

**Review by philipperolet — APPROVED** [review]
LGTM with 2 nits :+1:

File: sparkle/src/components/AssistantPreview2.tsx (hunk 2 above)
Comment (flvndvd) [reply sparkle/src/components/AssistantPreview2.tsx:30]: To stop propagating the event since we have a CTA on the parent div.

**Pull request merged**